### PR TITLE
 Add count to intercept method to specify number of times the request has to be intercepted

### PR DESCRIPTION
**Pull request by rparthas**

### CODE REVIEW

File: lib/taiko.js [hunk 1]
```diff
@@ -374,13 +374,15 @@ module.exports.switchTo = async targetUrl => {
  *
  * @param {string} requestUrl request URL to intercept
  * @param {function|Object} option action to be done after interception. For more examples refer to https://github.com/getgauge/taiko/issues/98#issuecomment-42024186
- *
+ * @param {number} count number of times the request has to be intercepted . Optional parameter
+ * 
  * @returns {Promise}
  */
-module.exports.intercept = async (requestUrl, option) => {
+module.exports.intercept = async (requestUrl, option, count) => {
```
Comment (Apoorva-GA) [inline lib/taiko.js]: It would be good to add an example with count.

**Comment by Apoorva-GA** [comment]
@rparthas Also please resolve the conflicts.

File: lib/taiko.js (hunk 1 above)
Comment (rparthas) [reply lib/taiko.js]: Sure @Apoorva-GA i will add them

**Comment by rparthas** [comment]
@Apoorva-GA  I could see one of the checks failed. Is there possibility to rerun the ci tests ? I am running with node 10 and don't seem to get any error in local

Connection to runner with Pid 3613 lost. The runner probably quit unexpectedly. Inspect logs for potential reasons. Error : read tcp 127.0.0.1:40061->127.0.0.1:41316: use of closed network connection

**Comment by Apoorva-GA** [comment]
@rparthas Thanks for the PR..!!

**Pull request merged**